src/tree/contexts/ContextTreeItem.ts

```diff
@@ -10,15 +10,26 @@ import { getThemedIconPath, IconPath } from '../IconPath';
 import { LocalContextInfo } from "./LocalContextInfo";
 
 export class ContextTreeItem extends AzExtTreeItem {
-    public static contextValue: string = 'context';
-    public contextValue: string = ContextTreeItem.contextValue;
+    public static allContextRegExp: RegExp = /Context$/;
+    public static removableContextRegExp: RegExp = /^customContext$/i;
+
     private readonly _item: LocalContextInfo;
 
     public constructor(parent: AzExtParentTreeItem, item: LocalContextInfo) {
         super(parent);
         this._item = item;
     }
 
+    public get contextValue(): string {
+        if (/^default$/i.test(this.name)) {
```

Comment:
Turns out it is completely possible to create an additional Docker context named `DEfault` or something like this, even on Windows. Should we check `this.name === 'default'` instead?

Reply:
That should be fine. We also do this same comparison in `refreshDockerode`. I'll change both.